script.js

```diff
@@ -1,4 +1,122 @@
+/*
+- player 1 rolls two dice and choose the order of the dice
+- player 2 rolls two dice and choose the order of the dice
+- compare player 1 and player 2 number, state the winner
+- reset the game
+*/
+
+var gameState = "player1Roll";
+var player1RollRecords = [];
+var player2RollRecords = [];
+var player1PointsRecord = [];
+var player2PointsRecord = [];
+
+// roll dice function
+var rollDice = function () {
+  let randomDiceNum = Math.floor(Math.random() * 6) + 1;
+  //console.log(randomDiceNum);
+  return randomDiceNum;
+};
+
+// player1 choose value function
+var chooseNumberFunction1 = function (input) {
+  if (input != 1 && input != 2) {
+    outputMessage = `Player 1, invalid option. 
+      For option 1, please key 1.
+      For option 2, please key 2.`;
+  }
```

Comment:
make use of early return and just return immediately! 

also it would be ideal from a user experience pov if you can show them their dice rolls again in a scenario where they input an invalid option. reason being: if i enter not 1 or 2, i may have forgotten my dice rolls already. when im greeted with message, i have no idea anymore what my eventual score will be if i enter 1 or 2